node/integrationtest/yarn.js

```diff
@@ -19583,7 +19583,7 @@ $exports.store = store;
      * @returns {boolean} Returns `true` if `value` is a buffer, else `false`.
      * @example
      *
-     * _.isBuffer(new Buffer(2));
+     * _.isBuffer(Buffer.from(2));
```

Comment:
This is just a comment, but this would throw.

Reply:
I now replaced `Buffer.from(<integer>)` with `Buffer.alloc(<integer>)`.